### DISCUSSION TITLE
Is there a way to add throttling to a livewire component public function?

### DISCUSSION THREAD

**Discussion by sukhcha-in** [livewire/livewire#1525] [4. General]
Question: Is there a way to add throttling to a livewire component public function?

So as to prevent too many form submits in a short time i guess.

**Comment by ZebTheWizard** [comment]
I was curious about the same thing. I made a proposal here. https://github.com/livewire/livewire/discussions/1528

**Reply by sukhcha-in** [reply]
> I was curious about the same thing. I made a proposal here. #1528

That is a different thing, not related to throttling.
